README_LINUX.md

```diff
@@ -249,34 +243,34 @@ In some situations, it is preferable to install libraries and plugins
 not in the `lib` directory but in a suffixed one, e.g. `lib64`.
 In such a case you can set the cmake variable `LIB_SUFFIX`.
 For example if you wish to install into `lib64`:
-
-    cmake -DLIB_SUFFIX=64 ..
-
+```shell
+cmake -DLIB_SUFFIX=64 ..
+```
 ### Step 4: Build
 
 If CMake ran successfully without errors, you are ready to move on to building. You can freely alternate between building and setting CMake flags.
 
 After setting your CMake flags, just run
-
-    make
-
+```shell
+make
+```
 The `-j` option allows multiple jobs to be run simultaneously, which can improve compile times on machines with multiple cores. The optimal `-j` setting varies between machines, but a good rule of thumb is the number of cores plus one. For example, on a 4-core system, try `make -j5`.
 
 And to install, run
-
-    sudo make install
-
+```shell
+sudo make install
+```
 You will need to use `make install` if you are doing a user-wide installation, which is not the default.
 
 After installing for the first time, please run
-
-    sudo ldconfig
-
+```shell
+sudo ldconfig
+```
 To uninstall:
-
-    make uninstall
-
-(or `sudo make uninstall`).
+```shell
+make sudo uninstall
```

Comment:
I think this should be `sudo make uninstall` ?

Reply:
Oh, sorry! It was a typo, and corrected!